tests/utils/verification/volumeproperties.go

```diff
@@ -96,6 +96,12 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+// CheckVolumeAccessibility returns true if the given volume is accessible
+// from the specified VM; false otherwise.
+func CheckVolumeAccessibility(hostName string, volumeName string) bool {
+	return GetVolumeStatusHost(volumeName, hostName) != ""
```

Comment:
Yes. We have 2 different way to implement this:
1. Using "docker volume inspect --name" to get the status of given volume name
2. Using "docker volume ls" to get the list of all existing volumes and check if the name is in the list
I used #1 just because it's easier.